pybossa.js

```diff
@@ -318,14 +318,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     function _getNotificationMessage(userProgress, isEmptyTask){
         var quiz = userProgress.quiz;
         var config = quiz.config;
-        var projectCompleted = _projectCompleted(userProgress, quiz, isEmptyTask);
         var outOfGoldenTasks = _outOfGoldenTasks(quiz, config, isEmptyTask);
         var failedQuiz = _failedQuiz(quiz, config);
         var outOfNonGoldTask = _outOfNonGoldTask(isEmptyTask)
 
-        return _readOnly() || outOfGoldenTasks || outOfNonGoldTask || _inGoldMode(isEmptyTask) ||
-               failedQuiz || projectCompleted ||
```

Comment:
Just to confirm, does removing `projectCompleted` from this condition prevent the popup from displaying too soon (before the redirect takes place)? Then, after the redirect to the project home `_userProgress()` is called which shows the popup?

Reply:
Yes, moving `projectCompleted` out prevents the popup from displaying  before redirecting with a url parameter `?completed=true`.
On the other end, we check the url parameter and display the message. Reference: https://github.com/bloomberg/pybossa-default-theme/pull/387